src/main/java/Controllers/GameController.java

```diff
@@ -61,6 +64,41 @@ public void init(String guestTeamName, String hostTeamName, String guestScore, S
         events_table.getColumns().addAll(gameMinuteColumn, typeColumn, descriptionColumn);
         events_table.getSortOrder().add(gameMinuteColumn);
 
+        add_event_btn.setOnAction(event -> addEventToGame(gameID, gameDate));
+
+    }
+
+    private void addEventToGame(String gameID, LocalDate gameDate) {
+        //test if the game can be edited (24 hours after it end)
+        if (LocalDate.now().minusDays(1).isEqual(gameDate) || LocalDate.now().isEqual(gameDate)) {
```

Comment:
It suppose to be 5 hours.

Reply:
can't do 5 hours with our current implementation, need to change the entire DB for it, wanan do it?